src/media/js/main.js

```diff
@@ -61,7 +62,7 @@ define('main', [
     });
 
     z.page.on('navigate', function(e, url) {
-        if ([urls.reverse('login'), urls.reverse('fxa_authorize')].indexOf(url)) {
+        if ([urls.reverse('login'), urls.reverse('core/fxa_authorize')].indexOf(url)) {
```

Comment:
It's not about infinitely recursing. When logging in, the fxa_authorize check would fail since url is like ```/fxa-authorize/?state=blah``` and ```reverse('core/fxa_authorze')``` is just ```fxa-authorize```. This would result in the wrong URL being sent to the login endpoint